src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -154,21 +157,22 @@ export const VenueLandingPage: React.FunctionComponent<VenueLandingPageProps> =
               />
             </div>
           )}
-          {(!futureOrOngoingVenueEvents ||
-            futureOrOngoingVenueEvents.length === 0) && (
-            <button
-              className="btn btn-primary btn-block btn-centered"
-              onClick={onJoinClick}
-            >
-              Join the event
-              {(venue?.start_utc_seconds ?? 0) >
-                new Date().getTime() / 1000 && (
-                <span className="countdown">
-                  Begins in {getTimeBeforeParty(venue.start_utc_seconds)}
-                </span>
-              )}
-            </button>
-          )}
+          {!hasSecretForm &&
+            (!futureOrOngoingVenueEvents ||
+              futureOrOngoingVenueEvents.length === 0) && (
+              <button
+                className="btn btn-primary btn-block btn-centered"
+                onClick={onJoinClick}
+              >
+                Join the event
+                {(venue?.start_utc_seconds ?? 0) >
+                  new Date().getTime() / 1000 && (
```

Comment:
This condition should go into a constant

`const showCountdown = (venue?.start_utc_seconds ?? 0) > new Date().getTime() / 1000`